apps/docs/pages/guides/database/extensions/wrappers/overview.mdx

```diff
@@ -1,74 +1,121 @@
+
+- An external database, like Postgres or Firebase.
+- A remote data warehouse, like ClickHouse, BigQuery, or Snowflake.
+- An API, like Stripe or GitHub.
+
+It's possible to connect to multiple remote servers of the “same type”. For example, you can connect to two different Firebase projects within the same Supabase database.
+
+### Foreign Tables
+
+A table in your database which maps to some data inside a Remote Server.
+
+Examples:
+
+- An `analytics` table which maps to a table inside your data warehouse.
+- A `subscriptions` table which maps to your Stripe subscriptions.
+- A `collections` table which maps to a Firebase collection.
+
+Although a Foreign Table behaves like any other table, the data is not stored inside your database. The data remains inside the Remote Server.
+
+### ETL with Wrappers
+
+ETL stands for Extract, Transform, Load. It's an establised process for moving data from one system to another. For example, it's common to move data from a production database to a data warehouse.
+
+There are many popular ETL tools, such as [Fivetran](https://fivetran.com/) and [Airbyte](https://airbyte.io/).
+
+Wrappers provide an alternative to these tools, because you can simply move data from one table to another using SQL:
 
-{/* prettier-ignore */}
 ```sql
-select
-  customer_id
-  currency
-from
-   stripe.customers;
+-- Copy data from your production database to your
+-- data warehouse for the last 24 hours:
 
-    customer_id     | currency 
---------------------+-----------
- cus_MJiBtCqOF1Bb3F | usd      
-(1 row)
+insert into warehouse.analytics
+select * from public.analytics
+where ts > (now() - interval '1 DAY');
 ```
 
-## Usage
+This approach provides several benefits:
 
-### Enable the extension
+1. **Simplicity:** the Wrappers API is just SQL, so data engineers don't need to learn new tools and languages.
+1. **Save on time:** avoid setting up additional data pipelines.
+1. **Save on Data Engineering costs:** less infrastructure to be managed.
 
-<Tabs
-  scrollable
-  size="small"
-  type="underlined"
-  defaultActiveId="dashboard"
-  queryGroup="database-method"
->
-<TabPanel id="dashboard" label="Dashboard">
+On the flip-side, Wrappers are not as feature-rich as these tools, and they couple the ETL process to your database.
 
-1. Go to the [Database](https://supabase.com/dashboard/project/_/database/tables) page in the Dashboard.
-2. Click on **Extensions** in the sidebar.
-3. Search for "wrappers" and enable the extension.
+### On-demand ETL with Wrappers
 
-</TabPanel>
-<TabPanel id="sql" label="SQL">
+Supabase extends the ETL concept with real-time data access. Instead of moving gigabytes of data from one system to another before you can query it, you can instead query the data directly from the remote server. This additional option, "Query", extends the ETL process and is called [QETL](https://www.sciencedirect.com/science/article/abs/pii/S0169023X1730438X) (pronounced "kettle"): Query, Extract, Transform, Load.
```

Comment:
some inconsistency in capitalization of `remote server` on this page. I don't have a preference on styling, but pick one and stick with it